frontend/src/components/DashboardDescriptionListGroup.tsx

```diff
@@ -110,11 +116,17 @@ const DashboardDescriptionListGroup: React.FC<DashboardDescriptionListGroupProps
       ) : (
         <DescriptionListTerm>
           <Flex
-            spaceItems={{ default: 'spaceItemsSm' }}
+            spaceItems={{ default: 'spaceItemsNone' }}
             alignItems={{ default: 'alignItemsCenter' }}
           >
             <FlexItem>{title}</FlexItem>
-            {tooltip}
+            {popover && (
+              <Popover bodyContent={popover}>
+                <Button variant="plain" aria-label="More info">
+                  <OutlinedQuestionCircleIcon />
+                </Button>
```

Comment:
Should use `DashboardPopupIconButton` because it doesn't have the typical button padding.